scm/driver/github/webhook.go

```diff
@@ -182,6 +186,89 @@ func (s *webhookService) parsePullRequestHook(data []byte) (scm.Webhook, error)
 	return dst, nil
 }
 
+func (s *webhookService) parsePipelineHook(data []byte) (scm.Webhook, error) {
+	src := new(pipelineHook)
+	err := json.Unmarshal(data, src)
+	if err != nil {
+		return nil, err
+	}
+	dst, err := convertPipelineHook(src), nil
+	return dst, nil
+}
+
+func convertPipelineHook(src *pipelineHook) *scm.PipelineHook {
+	namespace, name := scm.Split(src.WorkflowRun.Repository.FullName)
```

Comment:
do you need to parse FullName?  can you instead use `WorkflowRun.Repository.Name` as the name, and `WorkflowRun.Repository.Owner.Login` for the namespace.

Reply:
Update the changes